pkg/v1/tarball/write.go

```diff
@@ -213,3 +249,38 @@ func writeTarEntry(tf *tar.Writer, path string, r io.Reader, size int64) error {
 	_, err := io.Copy(tf, r)
 	return err
 }
+
+type WriteOption func(*writeOptions) error
+type writeOptions struct {
+	updates chan<- v1.Update
+}
+
+func WithProgress(updates chan<- v1.Update) WriteOption {
```

Comment:
nit: add godoc

Reply:
Dang! How did I miss that? I must have forgotten to lint it. Adding now.